components/UserInfos/UserInfos.tsx

```diff
@@ -59,9 +60,16 @@ const UserInfos: React.FC<UserInfosProps> = ({ lsp3JSON }) => {
     setFirstLink(undefined);
   }, [lsp3JSON, address]);
 
+  const firstFourOfAddress = () => {
+    if (address) {
+      //firs two bytes of address
+      return address.slice(2, 6);
+    }
+  };
```

Comment:
you can put this in utils file or smtgh other than this component as you may need to use this function elsewhere